net-im/webex/webex-41.9.0.20091.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg rpm
+
+DESCRIPTION="Cisco video conferencing and online meeting software"
+HOMEPAGE="https://www.webex.com/"
+SRC_URI="https://binaries.webex.com/WebexDesktop-CentOS-Official-Package/Webex.rpm -> ${P}.rpm"
```

Comment:
This is very unfortunate. Isn't there anywhere where this package can be downloaded with a fixed version? For example, I can't even test this due to:
```
!!! Fetched file: webex-41.9.0.20091.1.rpm VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      239953332
!!! Expected: 232153632
```
The `RESTRICT="mirror"` makes it impossible for us to merge this PR like this. And it also makes it impossible for any individual to host it.

I see this package is last-rited due to this reason, not much we can do, other than blame upstream...